Whitelisted scars for vulpkanin

Zephyrpak opened the pull request:
<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->

## About the PR
<!-- What did you change? -->
Whitelisted the scars found in the scars.yml to Vulpkanin because literally every other humanoid was there.
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Because Vulpkanin can have scars on their body too. They deserve to have top surgery scars and all the other cool scars!
## Technical details
<!-- Summary of code changes for easier review. -->
yml editing.
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. -->
Scars my beloved
![image](https://github.com/user-attachments/assets/776e99dc-c669-4427-82e5-8c3ebe56d3c1)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelogs must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- tweak: Vulpkanin now have scars whitelisted to them.

Zephyrpak committed:
Whitelisted vulps for scars

Added vulps to all scar whitelists, changed comment format, 1984'd the fact that the chest scar was male only?? Women can't get chest scars ig :blunt:

deltanedas commented on Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml:
```diff
@@ -2,7 +2,7 @@
   bodyPart: Head
   markingCategory: Head
-  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi]  # Delta V - Felinid, Oni, Harpy, Rodentia, Feroxi
+  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin]  # DeltaV - Felinid-Oni-Harpy-Rodentia-Feroxi-Vulpkanin
```
just have it be Felinid-Vulpkanin for all of these theres no point to keeping this meme alive

deltanedas commented on Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml:
```diff
@@ -44,8 +44,8 @@
   markingCategory: Chest
-  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi] #Einstein Engines - Felinid, Oni - DeltaV - Feroxi
-  sexRestriction: [Male] # DeltaV: Splitting the scars and tattoos
+  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi, Vulpkanin] #Einstein Engines - Felinid-Oni - DeltaV - Feroxi-Vulpkanin
```
idk why these had EE comments to begin with felinid and oni were stolen from nyano do the same for these

Zephyrpak committed:
Killing the meme per the request

Took the comments outback and shot them. (more touch up and fixing)

deltanedas commented on Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml:
```diff
@@ -44,8 +44,8 @@
-  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi] #Einstein Engines - Felinid, Oni - DeltaV - Feroxi
-  sexRestriction: [Male] # DeltaV: Splitting the scars and tattoos
+  speciesRestriction: [Human, Dwarf, Felinid, Oni, Feroxi, Vulpkanin] # DeltaV - Felinid-Vulpkanin
+  #sexRestriction: [Male] # DeltaV -  Splitting the scars and tattoos
```
actually just remove this since it was added by deltav

Zephyrpak committed:
Another comment 1984

Exploded the sexRestriction comment that was added by DeltaV, and I commented out. Better exploded than commented out ig.

Signed-off-by: Zephyr <68482449+Zephyrpak@users.noreply.github.com>
